docker/fsroot/start-minimega.sh

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+
+/usr/share/openvswitch/scripts/ovs-ctl start 1>/var/log/minimega.log 2>/var/log/minimega.log
+if [ $? -ne 0 ]; then
+    echo "Failed to start Open vSwitch" 1> /var/log/minimega.log
+fi
+
+# Check if Minimega is already running
+if pgrep -f "/opt/minimega/bin/minimega" > /dev/null; then
+    echo "Minimega is already running. Exiting script." 1> /var/log/minimega.log
+    exit 0
+fi
```

Comment:
@GhostofGoes and @jacdavi do you think that minimega would benefit from these additions to the `start-minimega.sh` or do you think it would cause a loss of functionality?

Reply:
The primary logic change is exiting the when minimega is already running. We call this script to start minimega when clearing our testbed, but if minimega (and miniweb) are still running, that isn't a useful action. Especially as trying to start multiple instances of minimega/miniweb seemed to cause issues (even with `-force True`).